database.php

```diff
@@ -0,0 +1,38 @@
+
+<?php
+// Step 1: Create a connection
+$db = new PDO('mysql:host=db; dbname=cocktails', 'root', 'password');
+
+// Step 2: Prepare a query
+$query = $db->prepare('SELECT * FROM `cocktails`;');
```

Comment:
I'd recommend this go into a function too. I'd also recommend that you pass the SQL query string in as a parameter to that function. That way you can run DB query with different queries without repeating code, as above!